scripts/obtain/dotnet-install.sh

```diff
@@ -84,6 +84,10 @@ get_current_os_name() {
                     echo "opensuse.13.2"
                     return 0
                     ;;
+                "opensuse.42.1")
```

Comment:
hmm this file shouldn't be checked in here. 

You'll need to make these changes in dotnet/cli in order for downstream consumers to take advantage of it.